nss-day-cohort-51/nss-day-cohort-51.github.io
# Ss feature hoverlinks 79

# Description - updated text animation, bg img in description, devcard buttons/icons on hover

## Type of change



# Testing Instructions
Fetch branch, serve, and look to see if carousel in dev list disappears on shrinking down to mobile size


# Checklist:
